app/assets/stylesheets/registrations.css

```diff
@@ -0,0 +1,61 @@
+.login-section, .signup-section {
+  text-align: center;
+}
+
+.login-title, .signup-title {
+  font-family: Abril Fatface;
```

Comment:
Add `sans-serif` at the end. This becomes the default value for the font in case the one you want is not found.
`font-family: Abril Fatface, sans-serif;`